src/inputs/index.js

```diff
@@ -21,6 +21,8 @@ export default class Input {
 
         this.store = store;
 
+        store.dispatch(ActionCreators.updateOutriggerUrl(`http://${outriggerUrl}`));
```

Comment:
idk, the fact that we're setting outriggerUrl into two different places, doesn't sit well with me. If this changes, someone will have to know that have to set it twice. At bare minimum we should probably be instantiate `OutriggerAPI` with `store.getState().outriggerUrl`, but would love to find a better solution.